src/main/java/ch/njol/skript/expressions/base/EventValueExpression.java

```diff
@@ -171,40 +171,40 @@ public boolean isSingle() {
 	}
 	
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
-		if (!debug || e == null)
+	public String toString(@Nullable Event event, boolean debug) {
+		if (!debug || event == null)
 			return "event-" + Classes.getSuperClassInfo(componentType).getName().toString(!single);
-		return Classes.getDebugMessage(getValue(e));
+		return Classes.getDebugMessage(getValue(event));
 	}
 	
-	@SuppressWarnings("unchecked")
 	@Override
 	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
-		Changer<? super T> ch = changer;
-		if (ch == null)
-			changer = ch = (Changer<? super T>) Classes.getSuperClassInfo(c).getChanger();
-		return ch == null ? null : ch.acceptChange(mode);
+	@SuppressWarnings("unchecked")
+	public Class<?>[] acceptChange(ChangeMode mode) {
+		Changer<? super T> changer = this.changer;
+		if (changer == null)
+			this.changer = changer = (Changer<? super T>) Classes.getSuperClassInfo(c).getChanger();
+		return changer == null ? null : changer.acceptChange(mode);
```

Comment:
This could be setup in the init or constructor rather than in the acceptChange method similar to your other pull request.